tests/serializers/test_python.py

```diff
@@ -61,6 +61,63 @@ class Parent(Object):
     )
 
 
+def test_and_parse_schema_with_description():
+    schema = {
+        "type": "object",
+        "title": "Parent",
+        "required": ["category"],
+        "properties": {
+            "category": {
+                "type": "object",
+                "title": "Category",
+                "properties": {
+                    "value": {
+                        "type": "string",
+                        "description": "value of category",
+                    }
+                },
+                "default": {"value": "none"},
+                "description": "category of parent",
+            },
+            "default": {"type": "string"},
+        },
+        "definitions": {
+            "other": {
+                "type": "object",
+                "title": "Other",
+                "properties": {"value": {"type": "integer"}},
+            }
+        },
+    }
+    assert serialize_python(*parse(schema)) == (
+        '''from statham.schema.constants import Maybe
+from statham.schema.elements import Integer, Object, String
+from statham.schema.property import Property
+
+
+class Other(Object):
+
+    value: Maybe[int] = Property(Integer())
+
+
+class Category(Object, default={'value': 'none'}, description='category of parent'):
+
+    """category of parent"""
+
+    value: Maybe[str] = Property(String(description='value of category'))
+    """value of category"""
```

Comment:
In contrast to my comment on `Object` classes above, I think for properties the `description=` keyword argument to the contained element is sufficient, and there shouldn't be a need to write a docstring under the property.

I'm less certain on this stance, but my rationale is roughly:

- In JSON Schema, the description is a description of the contained element type, _not_ in the context of the object which references it. There's nowhere to place such a description in JSON Schema.
- When converting from Python to JSON Schema, these docstrings cannot be used, because many classes may have differing properties with different docstrings pointing to the same underlying type.

Here's an example to illustrate:

```python

class Vector2D(Object):
    """A 2-dimensional vector."""
    x: float = Property(Number(), required=True)
    y: float = Property(Number(), required=True)

class KinematicParticle(Object):
    displacement: Vector2D = Property(Vector2D, required=True)
    """The current position of the kinematic particle relative to origin."""

    velocity: Vector2D = Property(Vector2D, required=True)
    """The current (directional) velocity of the kinematic particle."""

    acceleration: Vector2D = Property(Vector2D, required=True)
    """The current acceleration of the kinematic particle."""

```

In this example, the docstrings for each property are different and valid, but reference the same type. There is no place for description of a `property` in JSON Schema, so I think we shouldn't generate property docstrings from the schema.

Generating the in-line `description=` fields seems perfectly valid however.

Reply:
That sounds like a good argument. I remove the docstring generation from the argument.